src/ansys/turbogrid/core/multi_blade_row/multi_blade_row.py

```diff
@@ -0,0 +1,1130 @@
+# Copyright (c) 2023 ANSYS, Inc. All rights reserved
+"""Module for working on a multi blade-row turbomachinery case using PyTurboGrid instances in parallel."""
+
+from collections import OrderedDict
+from datetime import datetime as dt
+from datetime import timedelta as td
+import math
+from multiprocessing import Manager, Pool, Process, cpu_count
+import os
+import threading
+import time
+
+from ansys.turbogrid.api import pyturbogrid_core
+from ansys.turbogrid.api.CCL.ccl_object_db import CCLObjectDB
+from fabric import Connection
+from jinja2 import Environment, FileSystemLoader
+
+from ansys.turbogrid.core.launcher.launcher import launch_turbogrid
+from ansys.turbogrid.core.mesh_statistics import mesh_statistics
+import ansys.turbogrid.core.ndf_parser.ndf_parser as ndfp
+
+
```

Comment:
I would prefer "MultiBladeRow" as the class name; it is more consistent with the naming conventions and probably easier for a user to remember.